detekt.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide detekt rules for Kotlin projects
 homepageUrl=https://github.com/detekt/sonar-kotlin
-archivedVersions=2.0.0,2.2.0
-publicVersions=2.1.0,2.3.0
+archivedVersions=2.0.0,2.2.0,2.3.0
+publicVersions=2.1.0,2.4.0
 
 defaults.mavenGroupId=io.github.detekt
 defaults.mavenArtifactId=sonar-detekt
 
+2.4.0.description=Upgrade to detekt 1.18.1
+2.4.0.sqVersions=[7.9.4,LATEST]
+2.4.0.date=2021-10-31
+2.4.0.changelogUrl=https://github.com/detekt/sonar-kotlin/releases/tag/2.4.0
+2.4.0.downloadUrl=https://github.com/detekt/sonar-kotlin/releases/download/2.4.0/sonar-detekt-2.4.0.jar
+
 2.3.0.description=Upgrade to detekt 1.14.1
-2.3.0.sqVersions=[7.9.4,LATEST]
+2.3.0.sqVersions=[7.9.4,8.9.3]
```

Comment:
This range needs to end with the current version. Otherwise your users on 9.x versions won't be prompted to upgrade. 
To be explicit, since there's already a point version out for 9.2, I would end this with `9.2.*`